docs/release_notes/release_notes.md

```diff
@@ -14,7 +14,47 @@ For the full release notes, see the [Changelog](https://github.com/Katello/katel
 
 ## Features
 
+### Scenario Installer
+
+The installer has been updated to use the foreman-installer with a new feature known as scenarios. Scenarios provide a way to capture a precise set of installation options that can be re-used to install anything from a bare Foreman, to Katello to a Capsule. The initial scenarios being released are: Foreman, Katello and Caspule. For more information and examples, see the installation instructions.
+
+### OSTree Content
+
+Users can now manage OSTree content through Katello in the same ways that RPM content can be managed. This includes creating OSTree repositories, syncing those repositories and adding them to content views. Similar to RPM content, Atomic machines can be provisioned from local OSTree content. 
+
+### Lazy Sync
+
+### Docker V2
+
```

Comment:
The management of Docker content has been upgraded to support Docker v2.  This continues to involve the management of Docker content with repositories, syncing those repositories from an upstream registry and then publishing and promoting them through life cycle environments using content views.  Docker v2 does, however, provide some fundamental differences with Docker v1.  For example, rather than syncing Docker images and tags, the content synced will include manifests, tags and blobs.  In addition, with Docker v2, there is no longer the ability to upload Docker content to a repository; therefore, repository syncing is the means to get that content in to Katello.